spec/factories/users.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :user do
+    email { 'hal_jordan@gmail.com' }
```

Comment:
We don't want to be specifying email addresses ourselves, leave that to Faker. Your `random_user` factory should be your `user` factory. Anywhere you want to use a specific `email`, `password` or `password_confirmation` should be written manually in the spec for two reasons:

1) Test data should be random to prove that the code is working for various inputs (and, in turn, that the test isn't brittle)
2) If you wanted to specify something for any reason, that should be done in the test file as you should have a reason for doing so and that reason is best made obvious within the test